rules/azure/persistence_user_added_as_owner_for_azure_service_principal.toml

```diff
@@ -29,7 +29,7 @@ tags = ["Elastic", "Azure", "Continuous Monitoring", "SecOps", "Configuration Au
 type = "query"
 
 query = '''
-event.module:azure and event.dataset:azure.auditlogs and event.category:AuditLogs and azure.auditlogs.operation_name:"Add owner to service principal" and event.outcome:Success
+event.module:azure and event.dataset:azure.auditlogs and azure.auditlogs.operation_name:"Add owner to service principal" and event.outcome:Success
```

Comment:
This looks good, but can you update the name to include `Azure`

Reply:
nice, updated, verified all others have Azure in the name